docs/pim/create_product_types.md

```diff
@@ -23,8 +23,12 @@ Product code can have up to 64 characters. It can contain only letters, numbers,
 
 4\. Add fields that describe the product by dragging them from the **Field types** area to the **Field definitions** section, and then [configure them](../content_management/configure_ct_field_settings.md).
 
+ For the information that is intended to be displayed, use TextLine fields for short text, and RichText fields for longer text that may require formatting.
```

Comment:
What do you think about adding a link to the fieldtype reference here (https://doc.ibexa.co/en/4.6/content_management/field_types/field_type_reference/field_type_reference/) if someone wants to know more about the field types?